### PR TITLE
full audit

### PR DESCRIPTION
**Pull request by catlook**

### CODE REVIEW

**Comment by rackernexus** [comment]
Can one of the admins verify this patch?

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-29e9da6f1c/how-to/)

**Comment by catlook** [comment]
SME Regan Murley approved

File: content/cloud-files/manage-ttl-in-a-cloud-files-container.md [hunk 1]
```diff
@@ -16,16 +16,17 @@ how it works.
...
 container. This attribute and its value can be modified in the CDN
 through the Cloud Files user interface.
 
 New values take effect after the current TTL cycle is completed. The TTL
 can be any value between 15 minutes and 50 years. Use higher numbers for
 static content that doesn't change often, and use smaller numbers for
 content that changes more often. If you require a longer TTL, see the
-following blog post about using the API to set TTL: Extending TTL for
-Cloud Files CDN Users.
+following blog post about using the Cloud File Application Programmer
```
Comment (cyrichardson) [inline content/cloud-files/manage-ttl-in-a-cloud-files-container.md]: Cloud Files Application Programming Interface (API)

(Did a quick search on rest of how-to articles to see how this is most often presented.)

Comment (catlook) [reply content/cloud-files/manage-ttl-in-a-cloud-files-container.md]: doh!!

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-658972bda7/how-to/)

**Pull request merged**